docs/rabbitmq.config.example

```diff
@@ -572,8 +667,12 @@
 
    %% TCP/Socket options (as per the broker configuration).
    %%
-   %% {tcp_listen_options, [{backlog,   128},
-   %%                       {nodelay,   true}]}
+   %% {tcp_listen_options, [
```

Comment:
Any reason why there is no link to http://www.rabbitmq.com/networking.html here? It's a pretty extensive guide.